packages/epics/src/agreements/components/agreement-card.tsx

```diff
@@ -0,0 +1,125 @@
+import { Text } from '@radix-ui/themes';
+import { Card, Badge, Skeleton } from '@hypha-platform/ui';
+import Image from 'next/image';
+import { EyeOpenIcon, ChatBubbleIcon } from '@radix-ui/react-icons';
+
+type Creator = {
+  avatar?: string;
+  name?: string;
+  surname?: string;
+};
+
+type AgreementCardProps = {
+  creator?: Creator;
+  title?: string;
+  commitment?: number;
+  status?: string;
+  views?: number;
+  comments?: number;
+  isLoading?: boolean | undefined;
+};
+
+const statusBadge = (status: string, isLoading: boolean | undefined) => {
+  switch (status) {
+    case 'active':
+      return (
+        <Badge isLoading={isLoading} variant="positive">
+          Active
+        </Badge>
+      );
+    case 'voting':
+      return (
+        <Badge isLoading={isLoading} variant="warning">
+          On voting
+        </Badge>
+      );
+    case 'completed':
+      return (
+        <Badge isLoading={isLoading} variant="action">
+          Completed
+        </Badge>
+      );
+    case 'rejected':
+      return (
+        <Badge isLoading={isLoading} variant="destructive">
+          Rejected
+        </Badge>
+      );
+  }
+};
+
+export const AgreementCard: React.FC<AgreementCardProps> = ({
+  commitment,
+  status,
+  title,
+  creator,
+  views,
+  comments,
+  isLoading,
+}) => {
+  return (
+    <Card className="w-full h-full p-6 mb-2 flex">
+      {isLoading ? (
+        <Skeleton width={64} height={64} className="rounded-lg mr-3" />
+      ) : (
+        <Image
+          className="rounded-lg mr-3"
+          src={creator?.avatar ?? ''}
+          height={64}
+          width={64}
+          alt={
+            creator?.name && creator?.surname
+              ? `${creator.name} ${creator.surname}`
+              : 'Creator Avatar'
+          }
+        />
+      )}
+      <div className="flex justify-between items-center w-full">
+        <div className="flex flex-col">
+          <div className="flex gap-x-1">
+            <Badge isLoading={isLoading} variant="action">
+              Agreement
+            </Badge>
+            <Badge isLoading={isLoading} variant="actionOutline">
+              Recurring
+            </Badge>
+            <Badge isLoading={isLoading} variant="actionOutline">
+              {commitment}%
+            </Badge>
+            {status ? statusBadge(status, isLoading) : null}
```

Comment:
Together with the change above:
```suggestion
            <StatusBadge status={status} isLoading={isLoading} />
```